### DISCUSSION TITLE
Should we use `tauri-plugin` or `@tauri-apps/plugin` as the beginning of a new npm plugin's name?

### DISCUSSION THREAD

**Discussion by s3xysteak** [tauri-apps/tauri#9334] [General]
I found the [document](https://beta.tauri.app/guides/plugins/#naming-convention) recommends to use `@tauri-apps/plugin` as the beginning of a plugin's name. But in `www.npmjs.com` it seems a lot of plugin is using `tauri-plugin-` as the beginning. Is this a historical burden or some other convention?

**Comment by FabianLars** [comment]
Oh damn, that's indeed worded poorly. The next paragraph is a bit better but still kinda meh. We'll work on that, thanks for the feedback :)

Use `tauri-plugin-` or your own user's scope. You can't use a `@tauri-apps/` prefix/scope since we're the owner of that (assuming you want to publish it to npm).
